packages/admin/resources/views/pages/actions/select-action.blade.php

```diff
@@ -6,9 +6,10 @@
     <select
         id="{{ $getId() }}"
         wire:model="{{ $getName() }}"
-        {{ $attributes->class([
-            'text-gray-900 border-gray-300 invalid:text-gray-400 block w-full h-9 py-1 transition duration-75 rounded-lg shadow-sm focus:border-primary-600 focus:ring-1 focus:ring-inset focus:ring-primary-600',
-        ]) }}
+        @class([
```

Comment:
This can remain as `{{ $attributes->class([]) }}`, which supports the same syntax as `@class`.